frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx

```diff
@@ -1,11 +1,34 @@
 import './PageBoard.css';
+import { useEffect } from "react";
+
+import myApi from "../../../../../Url/Url";
 
 function PageBoard() {
+    useEffect(() => {
+        getTasksByIdProject("blabla")
+    }, []); 
+
+    const getTasksByIdProject =async (idProject:string) => {
```

Comment:
If you use it only in the useEffect please writ it inside the useEffect we dont want unpredict use cases

Reply:
i was change it in the future branch